src/renderer/services/wallet/balances.ts

```diff
@@ -340,19 +342,21 @@ export const createBalancesService = ({
   /**
    * List of `ChainBalances` for all available chains (order is important)
    */
-  const chainBalances$: ChainBalances$ = Rx.combineLatest(
-    filterEnabledChains({
-      THOR: [thorChainBalance$, thorLedgerChainBalance$],
-      BTC: [btcChainBalance$],
-      BCH: [bchChainBalance$],
-      ETH: [ethChainBalance$],
-      BNB: [bnbChainBalance$, bnbLedgerChainBalance$],
-      LTC: [litecoinBalance$]
-    })
-  ).pipe(
+  const chainBalances$: ChainBalances$ = FP.pipe(
+    Rx.combineLatest(
+      filterEnabledChains({
+        THOR: [thorChainBalance$, thorLedgerChainBalance$],
+        BTC: [btcChainBalance$],
+        BCH: [bchChainBalance$],
+        ETH: [ethChainBalance$],
+        BNB: [bnbChainBalance$, bnbLedgerChainBalance$],
+        LTC: [litecoinBalance$]
+      })
+    ),
     // we ignore all `ChainBalances` with state of `initial` balances
     // (e.g. a not connected Ledger )
-    RxOp.map(A.filter(({ balances }) => !RD.isInitial(balances)))
+    RxOp.map(A.filter(({ balances }) => !RD.isInitial(balances))),
+    RxOp.shareReplay(1)
```

Comment:
This `shareReplay` here avoid increasing numbers of requests for balances, which happened before by any new subscription to `chainBalances$`